include/hydra/backend/external_loop_closure_receiver.h

```diff
@@ -66,6 +71,12 @@ class ExternalLoopClosureReceiver {
     //! Maximum difference in seconds between a loop closure timestamp and the nearest
     //! agent pose. 0 disables checking time differences.
     double max_time_difference = 1.0;
+    //! Time window where a loop closure may block adding future loop closures
+    double lc_lockout_time = 5.0;
+    //! Threshold above which a loop closure constraining equivalent poses is considered
+    //! unique
+    double lc_min_pose_discrepancy = 2.0;
+
```

Comment:
(minor) I'd drop the LC prefix for the parameters (also the trailing extra newline)
```suggestion
    //! Time window where a loop closure may block adding future loop closures
    double lockout_time = 5.0;
    //! Threshold above which a loop closure constraining equivalent poses is considered
    //! unique
    double min_pose_discrepancy = 2.0;
```